fastlane/lib/fastlane/actions/ensure_no_debug_code.rb

```diff
@@ -22,6 +22,12 @@ def self.run(params)
 
         command << " --exclude #{params[:exclude]}" if params[:exclude]
 
+        if params[:exclude_dirs]
```

Comment:
This could be 
```ruby
Array(params[:exclude_dirs]).each do |dir|
  ...
```
Just a simple way to get rid of an `if` block, and allowing to iterate over the array even if the value is `nil`

```
Array(nil) # => []
```